src/fiskaltrust.ifPOS/v1/de/Models/TseInfo.cs

```diff
@@ -6,28 +6,39 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class TseInfo
     {
-        [DataMember(Order = 1)]
+        [DataMember(Order = 10)]
         public long MaxNumberOfClients { get; set; }
 
-        [DataMember(Order = 2)]
+        [DataMember(Order = 20)]
         public long CurrentNumberOfClients { get; set; }
 
-        [DataMember(Order = 3)]
+        [DataMember(Order = 30)]
+        public IEnumerable<string> CurrentClientIds { get; set; }
+
+        [DataMember(Order = 40)]
         public long MaxNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 4)]
+        [DataMember(Order = 50)]
         public long CurrentNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 5)]
+        /*
+        [DataMember(Order = 60)]
```

Comment:
Can we delete these?

Reply:
see https://github.com/fiskaltrust/middleware-interface-dotnet/pull/4#discussion_r417946263

it is cleaned up and hopefully accepted by @StefanKert  ;-)